README.md

```diff
@@ -50,7 +50,7 @@ Please note that this project is released with a [Contributor Code of Conduct](C
 
 ## License
 
-    Copyright 2019 Province of British Columbia
+    Copyright 2022 Province of British Columbia
```

Comment:
When updating copyrights, they should span a range. If you want to update this, it should be `2019-2022`